src/device-state.ts

```diff
@@ -400,8 +403,10 @@ export class DeviceState extends (EventEmitter as new () => DeviceStateEventEmit
 		log.debug('Starting periodic check for IP addresses');
 
 		await network.startIPAddressUpdate()(async (addresses) => {
+			const macAddress = await config.get('macAddress');
```

Comment:
You should be able to move this outside of the update loop, right? We don't expect this to change throughout runtime do we?

Reply:
yes we do - if a user add/removes a network interface, for example.